common/src/View/CreateSimpleBrushTool.cpp

```diff
@@ -33,7 +34,8 @@ namespace TrenchBroom {
 
         void CreateSimpleBrushTool::update(const vm::bbox3& bounds) {
             auto document = kdl::mem_lock(m_document);
-            const Model::BrushBuilder builder(document->world(), document->worldBounds());
+            auto game = document->game();
+            const Model::BrushBuilder builder(document->world(), document->worldBounds(), game->defaultFaceAttribs());
             updateBrush(builder.createCuboid(bounds, document->currentTextureName()));
```

Comment:
Default textures are not being used at all because `BrushBuilder::createCuboid` and `BrushBuilder::createBrush` does not have texture name as an optional parameter.